lib/pure/times.nim

```diff
@@ -700,46 +700,53 @@ else:
     result.inc tm.hour * secondsInHour
     result.inc tm.minute * 60
     result.inc tm.second
+  
+  when not defined(windows):
+    # Trick to find the real size of time_t
+    proc sizeof(typ: typedesc[posix.Time]): int =
+      {.emit: [result, " = sizeof(time_t);"].}
```

Comment:
Is there a better way to do this? The standard `sizeof` can't be used because `posix.Time` is always declared as `clong`, which might not be correct.

Reply:
I'm afraid no, but you need another workaround for the LLVM backend.